.travis.yml

```diff
@@ -6,11 +6,16 @@ os:
 language: rust
 
 rust:
-  - 1.31.1
+  - 1.35.0
   - stable
   - beta
   - nightly
 
+# There's a build problem on macOS, we hope this will be fixed.
+# See https://github.com/gtk-rs/cairo/issues/263
+env:
+  - PKG_CONFIG_PATH=/usr/local/opt/libffi/lib/pkgconfig
+
```

Comment:
needs a rebase?

Reply:
Probably should have done a rebase instead of a merge, but then it needs a force-push. In any case, it should turn out the same. Ah, git.